src/ukify/ukify.py

```diff
@@ -948,7 +948,15 @@ def make_uki(opts):
         ('.profile', opts.profile,    True ),
         ('.osrel',   opts.os_release, True ),
         ('.cmdline', opts.cmdline,    True ),
-        *(('.dtb', dtb, True) for dtb in opts.devicetree),
+    ]
+
+    # This ugly hack is an attempt to fix unit tests
```

Comment:
This should be solved differently. Maybe by ensuring that opts.devicetree is always a list even if only one DT is used?